### ISSUE TITLE
タスクリスト

### ISSUE THREAD

**Issue by SS1031** [SS1031/kaggle-shopee#1]
参考になりそうなノートブック

- [Pytorch Metric Learning Pipeline : Only Images](https://www.kaggle.com/tanulsingh077/pytorch-metric-learning-pipeline-only-images)
- [Metric Learning : Image + TFIDF Inference](https://www.kaggle.com/tanulsingh077/metric-learning-image-tfidf-inference?scriptVersionId=57597192)

**Comment by SS1031** [comment]
DistilBERT と TPU を使って学習するのに良さそうな資料

https://www.kaggle.com/xhlulu/jigsaw-tpu-distilbert-with-huggingface-and-keras/output
https://qiita.com/SPShota/items/7a37b4c534bea07dcf5e
https://www.kaggle.com/reppy4620/tfbertmodel-arcmargin-tpu-training
